takecare19/takecare19
# Edit resource

I accidentally deleted the prod resources while trying to update my local resources.... thankfully i fixed it 😅 

## testing instructions
- Click "edit" on a resource on the admin dashboard
- Make sure the form is prefilled with the correct information
- Update the info and submit, make sure the information has been updated
- Confirm the new resource form is still working
